src/plugins/image_display/ImageDisplay.cc

```diff
@@ -339,6 +342,60 @@ void ImageDisplay::UpdateFromLInt16()
   delete[] buffer;
 }
 
+/////////////////////////////////////////////////
+void ImageDisplay::UpdateFromLInt8()
```

Comment:
A lot of the code in this method looks very similar to the other `UpdateFrom*` methods in this class. It would be nice to re-factor the `UpdateFrom*` methods later when we have more time to remove the code duplication. Maybe we can add a TODO note about this? You can assign me to it if you'd like.

Reply:
added a todo note in c7b0fce and ticketed an issue https://github.com/ignitionrobotics/ign-gui/issues/177. Feel free to assign it to yourself once you get around to it, thanks.